### PR TITLE
Image Element

### PR DESCRIPTION
**Pull request by bajajcodes**
- Images are in three different options corner, rounded and sqaure shaped with fixed width and height of 200px.
- Images are made repsonsive using `max-width` and `height` property to scale down/up according to parent elment width.

https://user-images.githubusercontent.com/29247011/152290011-39127f90-ff88-44b9-89ea-33794f954028.mp4

### CODE REVIEW

File: css/elements/image.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@import "../utilities/variables.css";
+
+.img{
+    max-width: 100%;
+    height: auto;
+    object-fit: fill;
```
Comment (lannisterrr) [inline css/elements/image.css]: fill can squeeze the image , you can also use the property cover.

**Review by junaidshaikh-js — COMMENTED** [review]
Looks good to me!

**Review by FarhanMobashir — COMMENTED** [review]
All good ✅

File: css/elements/image.css (hunk 1 above)
Comment (bajajcodes) [reply css/elements/image.css]: Hey @lannisterrr I have change object-fit property to cover.

**Review by ZuberDunge — COMMENTED** [review]
Looks good to me

**Pull request merged**